include/grpc/impl/codegen/grpc_types.h

```diff
@@ -127,7 +127,11 @@ typedef struct {
     connection. Int valued. */
 #define GRPC_ARG_MAX_CONCURRENT_STREAMS "grpc.max_concurrent_streams"
 /** Maximum message length that the channel can receive. Int valued, bytes. */
-#define GRPC_ARG_MAX_MESSAGE_LENGTH "grpc.max_message_length"
+#define GRPC_ARG_MAX_RECEIVE_MESSAGE_LENGTH "grpc.max_receive_message_length"
```

Comment:
Question: Is it okay to change the value of this parameter?  I've added a `#define' to provide API backward-compatibility, but changing the string here affects ABI compatibility.  I'm not sure how big of an issue that is.